### ISSUE TITLE
Add a fun functionality to execute in a Console Application 

### ISSUE THREAD

**Issue by aherd2985** [aherd2985/UtilityBelt#42]
Please grab the latest version; it has been an active repo.

Please test within Visual Studio.

Be creative, have fun, happy coding!

**Comment by IrvinDominin** [comment]
I take it here: https://github.com/aherd2985/UtilityBelt/pull/45

**Comment by IrvinDominin** [comment]
Another here: https://github.com/aherd2985/UtilityBelt/pull/48

**Comment by IrvinDominin** [comment]
Another here: https://github.com/aherd2985/UtilityBelt/pull/51

**Comment by AlexGipp** [comment]
Can i add the functionallity to get Chuck norris jokes?

**Comment by aherd2985** [comment]
We have one that grabs Chuck Norris jokes from this api:
https://api.chucknorris.io/jokes/random

Do you have a different API for a Chuck Norris Ver 2.0 ?
________________________________

**Comment by AlexGipp** [comment]
Yeah just saw that as well @aherd2985, thats why i made a panda module instead :)

**Comment by AlexGipp** [comment]
Btw could i try to add unit tests for this project?

**Comment by aherd2985** [comment]
Please add unit tests, that would be awesome, thank you!

**Comment by AlexGipp** [comment]
Btw why is the structure of the project that weird? (no sln file for example)

**Comment by aherd2985** [comment]
It’s not needed; keeping the repo lean and trying to minimize merge issues between the contributors.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by AlexGipp** [comment]
Now there is an *.sln file, this is getting confusing.😂

**Comment by aherd2985** [comment]
I am sorry you are confused, but it is needed now to support multiple projects in one solution.

We recently had a great contribution that setup Unit Testing.

Please let me know if you have any questions, thanks!

**Comment by 1RedOne** [comment]
Will do this in #84 , adding Fox Facts

**Comment by AlexGipp** [comment]
@1RedOne what do you mean? I already implemented a random fox facts module: 
https://github.com/aherd2985/UtilityBelt/blob/master/Utilities/RandomFoxFact.cs

**Comment by 1RedOne** [comment]
LMAO, I missed it, nice job @AlexGipp

**Comment by c3sxr** [comment]
Hello guys,
I've included a new module for Yoda Translations [#92 ](https://github.com/aherd2985/UtilityBelt/pull/92)
Let me know what you guys think 😄

**Comment by strawberries73** [comment]
Hello, I am new and would like to contribute to this issue.

**Comment by aherd2985** [comment]
Please feel free to add any new utility to the project or join in on refactoring this into a dll output that can be imported and used everywhere, thank you for your interest!

**Comment by 1RedOne** [comment]
@strawberries73, if you can find some cool open Rest API, that would be a nice value-add to the project.  Some ideas:

* FaceBook / Twitter / Azure down-detector
* integrate with DownDetector.com?
* Grab newest X tweets about a hashtag

**Comment by IrvinDominin** [comment]
Added PostmanEcho https://github.com/aherd2985/UtilityBelt/pull/105 #hacktoberfestlove

**Comment by IrvinDominin** [comment]
Added RandomCommit text https://github.com/aherd2985/UtilityBelt/pull/106
